CMakeLists.txt

```diff
@@ -41,7 +41,8 @@ if(NOT DISABLE_WASM_BUILD)
     PREFIX ${PREFIX_DIR}/ui-wasm
     CONFIGURE_COMMAND
       ${EMCMAKE_COMMAND} ${CMAKE_COMMAND} -G "Unix Makefiles" -S ${CMAKE_SOURCE_DIR}/src/ui -B ${BUILD_DIR}/ui-wasm
-      -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE} -DCMAKE_MAKE_PROGRAM=${CMAKE_SOURCE_DIR}/emmake_make.sh
+      -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE} -DIS_UI_WASM_BUILD=ON
```

Comment:
could that not also be set if we did a ''manual'' emscripten build, which would include the utils target?

Reply:
We can do a manual emscipten build from the src/ui directory, which would not include the utils target. There it would actually be beneficial to add it, too, because currently that doesn't work. An emscripten build of the toplevel project will not work because the service will not compile for emscripten as it depends on zeromq, digitizer and other native dependencies.